services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/entity/MulticastGoalStateV2.java

```diff
@@ -65,6 +72,13 @@ public void setVpcIds(List<String> vpcIds) {
         this.vpcIds = vpcIds;
     }
 
+    public void addVpcId(String vpcId) {
+        if (this.vpcIds == null) {
+            vpcIds = new ArrayList<String>();
+        }
+        this.vpcIds.add(vpcId);
```

Comment:
@VanderChen What if `vpcId` is null?

Reply:
Should I check the input before usage or throw an exception here? Similar to the set method, it does not seem to check the validity of the input in the entity class.

Reply:
@VanderChen throw an exception is good enough for here and set method. You probably also need to handle the exception from the caller too.

Reply:
@cj-chung VpcIdNotFound() exception is added.